frontend/src/features/mapSideBar/SidebarContents/subforms/BuildingReviewPage.tsx

```diff
@@ -0,0 +1,284 @@
+import './BuildingReviewPage.scss';
+
+import {
+  FastSelect,
+  FastInput,
+  Input,
+  TextArea,
+  InputGroup,
+  SelectOptions,
+  FastDatePicker,
+  Check,
+  AutoCompleteText,
+  FastCurrencyInput,
+} from 'components/common/form';
+import React, { useCallback, useState } from 'react';
+import { Col, Container, Row } from 'react-bootstrap';
+import { useFormikContext } from 'formik';
+import { Label } from 'components/common/Label';
+import { FaEdit } from 'react-icons/fa';
+import { BuildingSvg } from 'components/common/Icons';
+import AddressForm from 'features/properties/components/forms/subforms/AddressForm';
+import { noop } from 'lodash';
+
+interface IReviewProps {
+  nameSpace?: string;
+  disabled?: boolean;
+  classifications: any;
+  predominateUses: SelectOptions;
+  constructionType: SelectOptions;
+  occupantTypes: SelectOptions;
+  agencies: any;
+}
+
+export const BuildingReviewPage: React.FC<any> = (props: IReviewProps) => {
+  const defaultEditValues = {
+    identification: true,
+    tenancy: true,
+    valuation: true,
+  };
+  const [editInfo, setEditInfo] = useState(defaultEditValues);
+  const withNameSpace: Function = useCallback(
+    (fieldName: string) => {
+      return props.nameSpace ? `${props.nameSpace}.${fieldName}` : fieldName;
+    },
+    [props.nameSpace],
+  );
+  const formikProps = useFormikContext();
+
+  return (
+    <Container className="building-review-section">
```

Comment:
Check both the land and building form sections to ensure they include the `Project No.` if it is provided.  You can see a sample of this in the latest code merged to `dev`.